docs/sphinx/advanced_configuration.rst

```diff
@@ -35,6 +36,11 @@ These arguments are explained in more detail below:
   This option enables support for NVIDIA GPUs. If Umpire is built without CUDA
   or ROCM support, then only the ``HOST`` allocator is available for use.
 
+* ``ENABLE_NUMA``
+  This option enables support for NUMA. The `strategy::NumaPolicy` is available
```

Comment:
if you add Doxygen, you will be able to use `` :class:`umpire::strategy::NumaPolicy` `` to link to it from here!

Reply:
Very cool! 👍